.travis.yml

```diff
@@ -3,5 +3,6 @@ sudo: false
 language: go
 go: tip
 script:
+    - go build -v ./...
     - go test -v ./storage/...
-    - go build -v ./management/...
+    - go test -v ./management/...
```

Comment:
the ./core dir also has some tests... not all of them succeed... didn't look why they fail.